tests/runner.py

```diff
@@ -1100,24 +1100,32 @@ def do_GET(self):
         self.end_headers()
         self.wfile.write(open(path_from_root('tests', 'browser_harness.html'), 'rb').read())
       elif 'report_' in self.path:
+        # for debugging, tests may encode the result and their own location as result|location
```

Comment:
what does 'location' mean in this context?

Reply:
I'll clarify in a comment - I meant `window.location`, i.e., the url it is being sent from.